src/transformers/tokenization_utils_base.py

```diff
@@ -2163,6 +2163,7 @@ def convert_added_tokens(obj: Union[AddedToken, Any], add_type_field=True):
         # remove private information
         if "name_or_path" in tokenizer_config:
             tokenizer_config.pop("name_or_path")
+            tokenizer_config.pop("special_tokens_map_file")
```

Comment:
```suggestion
            tokenizer_config.pop("special_tokens_map_file", None)
```
To avoid a hard error if the key is not there for some reason.